src/client/app/+home/home.component.ts

```diff
@@ -13,45 +17,49 @@ import { GraphPreviewListService } from '../shared/index';
 
 export class HomeComponent implements OnInit {
 
-  newGraphPreview: string = '';
-  errorMessage: string;
-  graphPreviews: any[] = [];
+  userLoggedIn: boolean = false;
+  graphPreviews: DriveFile[] = [];
 
   /**
    * Creates an instance of the HomeComponent with the injected
    * GraphPreviewListService.
    *
-   * @param {GraphPreviewListService} GraphPreviewListService - The injected GraphPreviewListService.
+   * @param googleRealtimeService
```

Comment:
Yeah I'd like to use JSdoc, following [Google Style](https://google.github.io/styleguide/jsguide.html#jsdoc).